site/content/docs/user/private-registries.md

```diff
@@ -90,6 +91,24 @@ rm $HOME/.docker/config.json
 [-f $HOME/.docker/config.json.host] && mv $HOME/.docker/config.json.host $HOME/.docker/config.json
 ```
 
+### ... with Service Accounts
```

Comment:
possibly expand the `...` here?

Reply:
Perhaps `Add Credentials Programmatically with Access Token` and `Add Credentials Programmatically with Service Account`? A little redundant, but we sadly can't nest further than an h3.

Reply:
ok, i see the nesting now.

IMO it should be:

> ## Add Credentials to the Nodes
> ### Mount a Config File to Each Node
> ### Use an  Access Token
> ### Use a Service Account

Reply:
> ...  but we sadly can't nest further than an h3.

Not that we should but, why can't we?

Reply:
@BenTheElder I might actually be wrong. Some markdown interpreters don't like going being h3; I think @neolit123 has the right nesting idea.

Will make changes soon :tm:

Reply:
FWIW we use [hugo](https://gohugo.io/) and you can view the rendered version at https://deploy-preview-796--k8s-kind.netlify.com/ but @neolit123's suggestion SGTM